packages/common/src/store/ui/buy-sell/useGroupCoinPairs.ts

```diff
@@ -33,6 +35,19 @@ export const useGroupCoinPairs = (coins?: UserCoin[], singleColumn = false) => {
     // Group coins for responsive layout
     const coinPairs: CoinPairItem[][] = []
 
+    // If no coins after filtering, show audio coin card
```

Comment:
hang on, shouldn't we just change the filter conditions above to always return true if `mint == env.WAUDIO_MINT_ADDRESS`?

Reply:
we still need an extra condition there tho, so adding